apps/condo/domains/ticket/access/UserFavoriteTicket.js

```diff
@@ -21,8 +21,8 @@ async function canReadUserFavoriteTickets ({ authentication: { item: user } }) {
         user: { id: user.id },
         organization: {
             OR: [
-                queryOrganizationEmployeeFor(user.id),
```

Comment:
same
1) Is permission check necessary here?
2) If necessary, is it necessary to check this right when creating/updating?